index.html

```diff
@@ -8,7 +8,9 @@
   <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v2.9.2/mapbox-gl.css' rel='stylesheet' />
   <link rel="stylesheet"
     href="https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v5.0.0/mapbox-gl-geocoder.css" type="text/css">
-  <link rel="stylesheet" href="css/font-awesome/css/font-awesome.min.css" />
+  <link rel="stylesheet" href="css/fontawesome/css/fontawesome.min.css" />
+  <link rel="stylesheet" href="css/fontawesome/css/brands.min.css" />
```

Comment:
@chriswhong For some reason I'm still getting a 404 on the `brands` file even after running `make` successfully. It's not a blocker here though - the icons in use still work great.